draft-ietf-quic-transport.md

```diff
@@ -1462,13 +1479,13 @@ connection if the integrity check fails with a PROTOCOL_VIOLATION error code.
 
 ## Connection Migration {#migration}
 
-QUIC connections are identified by their 64-bit Connection ID.  QUIC's
-consistent connection ID allows connections to survive changes to the client's
-IP and/or port, such as those caused by client or server migrating to a new
-network.  Connection migration allows a client to retain any shared state with a
-connection when they move networks.  This includes state that can be hard to
-recover such as outstanding requests, which might otherwise be lost with no easy
-way to retry them.
+QUIC connections are identified by their 136-bit Connection ID or whatever part
```

Comment:
Along the lines of being explicit about which side we're truncating, maybe call this a "prefix" rather than a "part"?